generators/client/templates/angular/src/main/webapp/content/scss/vendor.scss.ejs

```diff
@@ -23,9 +23,16 @@ put Sass variables here:
 eg $input-color: red;
 ****************************/
 // Override Boostrap variables
+<%_ if (clientTheme === 'none') { _%>
 @import "bootstrap-variables";
-// Import Bootstrap source files from node_modules
+<%_ } else {_%>
+@import "~bootswatch/dist/<%= clientTheme %>/variables";
+<%_ } _%>// Import Bootstrap source files from node_modules
 @import '~bootstrap/scss/bootstrap';
```

Comment:
can you cross check if this is required when user chooses custom theme. Generally customized themes also import bootstrap and you don't have to do this yourself

Reply:
No we need bootstrap also: https://github.com/thomaspark/bootswatch

Reply:
ok. I hope none of theme functionality depends on bootstrap javascript that we don't include